README.md

```diff
@@ -1,4 +1,5 @@
 # CP-8 Cli
+CP-8 is a ruby tool which aids in interacting with github by opening draft PRs, creating suggestion branches, opening CI for a branch etc.
```

Comment:
What do you think about this?

```
CP8 is a command line tool which aids in interacting with GitHub.

CP8 helps you open draft pull requests, suggests branch names and opens your CI for a branch.
```

Reply:
Thanks, this looks great. I've updated the PR.